.github/workflows/docker.yml

```diff
@@ -25,6 +25,7 @@ on:
            "ubuntu-jammy",
            "ubuntu-lunar",
            "ubuntu-mantic",
+           "ubuntu-noble",
```

Comment:
Please remove some old systems to keep the maintenance effort and the cpu time (relatively) constant.

Reply:
No, we will continue to test what is supported.

Reply:
Do you mean supported by SageMath or the Ubuntu organization?

Reply:
![image](https://github.com/sagemath/sage/assets/373765/ab2e1767-4ff3-4a86-ba81-65a30a2898cd)

If the latter, I am not sure we should worry about "pro support". But maybe we have a policy about that that I am not aware of?

Reply:
@saraedum I mean supported by SageMath.

Reply:
@saraedum for reference, what is supported by SageMath and why we drop support is tracked in #32074.